.github/workflows/fuzzers.yml

```diff
@@ -21,17 +21,17 @@ jobs:
           - aflcc
           - aflfast
           - aflplusplus
-          - aflplusplus_optimal
-          - aflplusplus_eclipser
+          # Disabled due to segfaults and timeouts.
+          # See https://github.com/google/fuzzbench/issues/1073.
+          # - aflplusplus_optimal
+          # - aflplusplus_eclipser
           - aflsmart
           - entropic
           - fairfuzz
-          - fastcgs_lm
           - honggfuzz
           - lafintel
           - klee
           - libfuzzer
-          - manul
```

Comment:
Why remove manul? Because it isn't so actively developed?

Reply:
Yes, and dont remember seeing interesting results. We need to scale benchmarks, so thinking to keep CI lean on what we periodically benchmark against.